firestore/firestore_snippets/solution_counters.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"cloud.google.com/go/firestore"
+	"google.golang.org/api/iterator"
+)
+
+// [START fs_counter_classes]
+// counters/${ID}
```

Comment:
Samples should be idiomatic for each language. For Go, comments follow a specific format (https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences). Also, need a newline between the region tag and the comment. Otherwise, it shows up in GoDoc.

Reply:
Done